client/extensions/woocommerce/state/ui/shipping/zones/methods/selectors.js

```diff
@@ -12,8 +12,10 @@ import {
 	isNil,
 	map,
 	merge,
```

Comment:
Is `merge` still used?

Reply:
It is used in a few places and I preferred to avoid changing them, as they are not involved in the shipping clases field. I am not sure what the general strategy is, but in this case I choose the "if it aint broke dont fix it" mantra.

Should I do it?